script/queries/steven_new.rb

```diff
@@ -96,8 +96,11 @@ def bad_eg_id(eg_id)
 end
 
 upload_to_s3 = Aws::S3Storage.new
-uri = upload_to_s3.save(filename,"reports")
-upload_to_s3.publish_to_sftp(filename,"Legacy::PushGlueEnrollmentReport",uri)
+bucket_name = upload_to_s3.env_bucket_for_glue_report
+uri = upload_to_s3.env_uri(bucket_name)
```

Comment:
this we need because we can not call instance method because bucket name gets build we do not want that.